src/navigation/constants.js

```diff
@@ -25,6 +25,7 @@ export const ROUTES = {
   CUSTOMER_INVOICES: 'customerInvoices',
 
   SUPPLIER_INVOICE: 'supplierInvoice',
+  SUPPLIER_INVOICE_WITH_PRICES: 'supplierInvoiceWithPrices',
```

Comment:
This isn't used anywhere.. should the `SupplierInvoicePage` be updated to use the CONSTANT?
Or is that part of another issue?

Reply:
Good idea. 554d700c9f416409d6c90af176f5a550a05f418a.

Reply:
Good idea. 554d700c9f416409d6c90af176f5a550a05f418a.